backend/users/views.py

```diff
@@ -0,0 +1,33 @@
+from django.http import HttpResponse,JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.core import serializers
+from .models.models import UserModel
+from rest_framework.views import APIView
+import json
+
+class User(APIView):
+    def get(self, request, format=None):
+        search = request.GET.get("search","")
+        users = []
+        if( search != ""):
+            users = serializers.serialize("json", UserModel.objects.filter(name__contains=search))
+        else:
+            users = serializers.serialize("json", UserModel.objects.all())
+        return HttpResponse(users, content_type='application/json')
+    def post(self, request, format=None):
+        print("Body = {}".format(str(request.data)))
+        email = request.POST.get("email", "")
+        password = request.POST.get("password", "")
+        name = request.POST.get("name", "")
+        if( email == ""):
+            return HttpResponse("Preencha o campo de email",status=400)
+        if( password == ""):
+            return HttpResponse("Preencha o campo de senha",status=400)
+        if( name == ""):
+            return HttpResponse("Preencha o campo de nome",status=400)
```

Comment:
Should be error 500

Reply:
The status code 500 = Internal server error, using for Exceptions or any error related the programing. But status code 400 = Bad Request, using for any error caused by client, in this case, not send the params in body.